terraform/db.tf

```diff
@@ -31,7 +31,7 @@ resource "aws_rds_cluster" "pgsql" {
 
   # 2AM-4AM PST
   preferred_backup_window = "09:00-11:00"
-  backup_retention_period = var.target_env == "prod" ? 14 : 3
+  backup_retention_period = var.target_env == "dev" ? 0: 14
```

Comment:
@chelsea-EYDS just for your attention, in case you look to spin up the tools env next week - this will also need tweaking 😄